### PR TITLE
operator->

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**

### CODE REVIEW

File: quantiles/include/quantiles_sketch.hpp [hunk 1]
```diff
@@ -582,11 +582,13 @@ class quantiles_sketch {
 template<typename T, typename C, typename A>
 class quantiles_sketch<T, C, A>::const_iterator: public std::iterator<std::input_iterator_tag, T> {
 public:
+  using value_type = std::pair<const T&, const uint64_t>;
```
Comment (jmalkin) [inline quantiles/include/quantiles_sketch.hpp:585]: same question on naming

File: req/include/req_sketch.hpp [hunk 2]
```diff
@@ -401,11 +401,13 @@ class req_sketch {
 template<typename T, typename C, typename A>
 class req_sketch<T, C, A>::const_iterator: public std::iterator<std::input_iterator_tag, T> {
 public:
+  using value_type = std::pair<const T&, const uint64_t>;
```
Comment (jmalkin) [inline req/include/req_sketch.hpp:404]: and once more

File: kll/include/kll_sketch.hpp [hunk 3]
```diff
@@ -588,12 +588,14 @@ class kll_sketch {
 template<typename T, typename C, typename A>
 class kll_sketch<T, C, A>::const_iterator: public std::iterator<std::input_iterator_tag, T> {
 public:
+  using value_type = std::pair<const T&, const uint64_t>;
```
[the remark this answers is not in the document]
Comment (AlexanderSaydakov) [reply kll/include/kll_sketch.hpp:591]: this is a standard name: https://cplusplus.com/reference/iterator/iterator_traits/

[the remark this answers is not in the document]
Comment (jmalkin) [reply kll/include/kll_sketch.hpp:591]: Kind of unfortunate in this case, but oh well.

**Review by jmalkin — APPROVED** [review]
LGTM

**Pull request merged**